Timer.php

```diff
@@ -163,7 +193,13 @@ public static function del($timer_id)
             return self::$_event->del($timer_id, EventInterface::EV_TIMER);
         }
 
-        return false;
+        foreach(self::$_tasks as $run_time => $task_data) 
+        {
+            if(array_key_exists($timer_id, $task_data))     unset(self::$_tasks[$run_time][$timer_id]);
+            if(array_key_exists($timer_id, self::$_status)) unset(self::$_status[$timer_id]);
```

Comment:
I think 
```php
if(array_key_exists($timer_id, self::$_status)) unset(self::$_status[$timer_id]);
```
should move to outside of foreach.

Reply:
yeah, u r right, tks.